### DISCUSSION TITLE
Make the popup editable

### DISCUSSION THREAD

**Discussion by mtaishiro** [manugarg/pagenotes-issues#68] [Ideas]
I love the chrome extension so far, including the popup feature, but I would like the popup to be editable directly instead of just being able to view it.
This is because I am currently using arc browser, and have a vertical toolbar where I have to move the mouse to the very left in order to make a change.

**Comment by manugarg** [comment]
I'm sorry to disappoint, but auto-popup was supposed to be just read-only. We'd like to avoid building two very parallel, very similar ways to edit notes. I don't use Arc myself but perhaps sidepanel will be a better interface for such a browser? https://github.com/manugarg/pagenotes-issues/issues/74
